src/lib/protocol/README.md

```diff
@@ -742,3 +788,18 @@ datagram payload and its byte size.
 
 If *pointer* and *length* are supplied then *length* bytes starting from
 *pointer* are appended to the datagram's payload.
+
+— Method **datagram:data**
+
+Returns the location and size of the buffer of the underlying packet.
+This is a shortcut to *datagram:packet* followed by calls to
+*packet.data* and *pakcet.length*.
```

Comment:
There's a typo here:  *pakcet*
Please use `M-x ispell` (or equivalent if you are not using Emacs), I know its a chore but its the only way I know of to avoid this systematically.